tracpro/baseline/views.py

```diff
@@ -74,18 +74,40 @@ def get_context_data(self, **kwargs):
 
             region = self.request.region
 
-            baseline_dict = self.object.get_baseline(region=region)
-            context['baseline_dict'] = baseline_dict
+            baseline_dict, baseline_dates = self.object.get_baseline(region=region)
 
             follow_ups, dates = self.object.get_follow_up(region=region)
 
+            """
```

Comment:
I'm not sure that I have ever seen block comments anywhere except for a docstring, but I don't have any good sources telling me that's the way it should be. Do you have a reason or reference for doing this?